indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/helpers/GenesisHelper.kt

```diff
@@ -0,0 +1,55 @@
+package com.luxoft.blockchainlab.hyperledger.indy.helpers
+
+import java.io.File
+import java.io.FileNotFoundException
+import java.nio.file.Path
+
+
+/**
+ * Utilities for genesis file management
+ */
+object GenesisHelper {
+
+    /**
+     * Get already existing genesis file
+     *
+     * Checks if genesis file exists and returns [File] object.
+     *
+     * @param genesisPath: String - path to genesis file
+     * @return: [File] - genesis file
+     * @throws FileNotFoundException - if genesis file not found
+     */
+    @Throws(FileNotFoundException::class)
+    fun getGenesis(genesisPath: String): File {
+        val genesisFile = File(genesisPath)
+
+        // making this check here for better debugging
+        if (!genesisFile.exists())
+            throw FileNotFoundException("Genesis file not found at $genesisPath")
+
+        return genesisFile
+    }
+
+    /**
+     * Get genesis file if it doesn't exist in system yet
```

Comment:
Description may be misleading. May be we should call it createGenesis, because it actually does this)